Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -389,7 +389,10 @@ class OverviewViewController: UIViewController {
     }
     
     @objc func handleBackupWallet() {
```

Comment:
This function name is wrong, there is no "backup wallet" feature, it's backup wallet seed. I understand this wasn't named in your PR, but please correct.